File: 3379-ScoreOfAString/3379-ScoreOfAString.java
// Last updated: 6/4/2026, 10:19:11 PM
class Solution {
    public int scoreOfString(String s) {
        int a=0;
        for(int i=1;i<s.length();++i)
        a+=Math.abs(s.charAt(i)-s.charAt(i-1));
        return a;
        
    }
}

File: 3379-ScoreOfAString/3379-ScoreOfAString.py
# Last updated: 6/4/2026, 10:19:01 PM
class Solution:
    def scoreOfString(self, s: str) -> int:
        score =0
        for i in range(1,len(s)):
         score =score + abs(ord(s[i])-ord(s[i-1]))
        return score